lms/static/scripts/frontend_apps/components/test/BookList-test.js

```diff
@@ -0,0 +1,68 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import BookList from '../BookList';
+
+describe('BookList', () => {
+  const bookData = [
+    {
+      id: 'test-book',
+      title: 'Test book',
+      cover_image: 'https://example.com/test.jpg',
+    },
+  ];
+
+  const noop = () => {};
+  const renderBookList = (props = {}) =>
+    mount(
+      <BookList
+        books={bookData}
+        selectedBook={null}
+        onSelectBook={noop}
+        onUseBook={noop}
+        {...props}
+      />
+    );
+
+  it('renders book titles', () => {
+    const bookList = renderBookList();
+    assert.equal(bookList.find('tbody > tr').length, bookData.length);
+    assert.include(bookList.html(), '<td>Test book</td>');
+  });
+
+  [true, false].forEach(isLoading => {
+    it('shows loading indicator if books are being fetched', () => {
+      const bookList = renderBookList({ isLoading });
+      assert.equal(bookList.find('Table').prop('isLoading'), isLoading);
+    });
```

Comment:
Ah! The test description here had me ready to comment "Is this testing `Table` functionality?" But then I realized the test itself is appropriate for `Booklist` (making sure that the `isLoading` prop is set correctly). I wonder if adjusting the test description would help, as the test is not about the loading indicator display (which is a responsibility of `Table`), though that is the manifest result.

(There is a similar test for `ChapterList`)

Reply:
Ah OK. I'll rephrase this.